core/templates/pages/landing-pages/stewards-landing-page/stewards-landing-page.component.spec.ts

```diff
@@ -21,6 +21,8 @@ import { WindowDimensionsService } from
   'services/contextual/window-dimensions.service';
 import { UrlService } from
   'services/contextual/url.service';
+import { of } from 'rxjs';
+
```

Comment:
I don't think this extra new line is required.

Reply:
Done